modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -24,25 +23,50 @@ and the `bmc` parameter for the `install-config.yaml` file.
 | `UEFI`
 | The boot mode for a node. Options are `legacy`, `UEFI`, and `UEFISecureBoot`. If `bootMode` is not set, Ironic sets it while inspecting the node.
 
-| `bootstrapExternalStaticDNS`
+a|
+----
+platform:
+  baremetal:
+    additionalNTPServers:
+    - <ip_address_or_domain_name>
+----
```

Comment:
additionalNTPservers is not a required parameter. It can be used when the hosts for the cluster install have their system time out of sync.

Reply:
I've modiefied the docs to indicate that the setting is optional with the cluster host clocks are out of synchronization. Can you let me know if this meets the requirements of HCIDOCS-373?